tsbridge/config.go

```diff
@@ -2,11 +2,14 @@ package tsbridge
 
 import (
 	"time"
+
+	"github.com/google/ts-bridge/stackdriver"
 )
 
 // Config holds all the ConfigOptions
 type Config struct {
-	Options ConfigOptions
+	Options      ConfigOptions
+	Dependencies ExternalDependencies
```

Comment:
Nit: `ExternalDependencies` is a bit confusing as it leads me to think this is somehow related to library load or something. Maybe go with Initializers or something?

Reply:
See recommendation from nerdinary, went with UpdateMetrics, wdyt?

Reply:
Moved this out of config and went with Metrics struct name instead.